src/app/charts/content/cminfo.yaml

```diff
@@ -28,3 +28,17 @@ s3:
     region: us-east-1
   aliases:
     - minio
+
+box:
+  url: https://box.com
+  ui: https://box.com
+  users:
+    admin: box.admin@armedia.com
+    main: box.admin@armedia.com
+  groups:
+    admin:
+      - box-administrators
+  settings:
+    private-key:
+      path: "/app/home/.content-private-key"
```

Comment:
This setting is probably best moved to the `core` chart's `cminfo.yaml`, since that chart will be the one to consume this value. Putting this into the connection secrets serves no purpose since each pod that mounts it can decide where they expose the license file, which may or may not be this same path.